src/lib.rs

```diff
@@ -189,14 +189,14 @@ pub struct Context<A: Actor + ?Sized> {
 /// whether to spawn the Actor into its own thread or block
 /// on the current calling thread.
 #[must_use = "You must call .spawn() or .block_on() to run this actor"]
-pub struct SpawnBuilder<'a, A: Actor> {
+pub struct SpawnBuilder<'a, A: Actor, F: FnOnce() -> A> {
     system: &'a mut System,
     capacity: Option<usize>,
     addr: Option<Addr<A>>,
-    factory: Box<dyn FnOnce() -> A + Send + 'static>,
+    factory: Box<F>,
```

Comment:
```suggestion
    factory: F,
```
I was unsure whether FnOnce() is Sized or not, but apparently yes, this seems to work for me.